src/App.js

```diff
@@ -139,6 +140,7 @@ const App = ({ locale, location, dependencies: { appSessionRef, sessionManager }
                 to="/forgot-password?lang=es"
               />
               <RedirectWithQuery exact from="/forgot-password" to="/login/reset-password" />
+              <PrivateRoute path={['/invoices-list']} exact component={InvoicesList} />
```

Comment:
Minor: Maybe you could move this line to keep all PrivateRoute elements together.

Reply:
👍